desktop/src/apps/ProjectsApp/ProjectsApp.module.css

```
@@ -0,0 +1,530 @@
+}
+.codeView {
+  font-family: "SF Mono", ui-monospace, Menlo, monospace;
+  font-size: 11.5px;
+  line-height: 1.65;
+  background: var(--color-shell-bg);
+  border: 1px solid var(--color-shell-border);
+  border-radius: 14px;
+  padding: 16px 18px;
+  color: var(--color-shell-text-secondary);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)